openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1507,7 +1507,96 @@ It is up to the Credential Issuer whether to update both the signature and the c
+to the Wallet. It should be made clear to the End-User what information is being included in the
+Credential(s) and for what purpose.
+
+## Minimum Disclosure
+
+To ensure minimum disclosure and prevent Verifiers from obtaining claims unnecessary for the
+transaction at hand, when issuing Credentials that are intended to be created once and then used a
+number of times by the End-User, the Credential Issuers and the Wallets SHOULD implement
+Credential formats that support selective disclosure, or consider issuing a separate Credential
+for each user claim.
+
+## Storage of the Credentials
+
+To prevent a leak of End-User data, especially when it is signed, which risks revealing private data of End-Users to
+third parties, systems implementing this specification SHOULD be designed to minimize the amount
+of End-User data that is stored. All involved parties SHOULD store Verifiable Credentials
+containing privacy-sensitive data only for as long as needed, including in log files. Any logging of
+End-User data should be carefully considered as to whether it is necessary at all. The time logs are retained
+for should be minimized.
+
+After Issuance, Credential Issuers SHOULD NOT store the Issuer-signed Credentials if they
+contain privacy-sensitive data. Wallets SHOULD store Credentials only in encrypted form, and,
+wherever possible, use hardware-backed encryption. Credentials that are no longer valid to present should be deleted as soon as
+possible.
+
+## Correlation 
+
+### Unique Values Encoded in the Credential
+
+Issuance/presentation or two presentation sessions by the same End-User can be linked on the basis of
+unique values encoded in the Credential (End-User claims, identifiers, Issuer signature, etc.) either by colluding Issuer/Verifier or Verifier/Verifier pairs, or by the same Verifier.
+
+To prevent these types of correlation, Credential Issuers and Wallets SHOULD use
+methods, including but not limited to the following ones:
+
+* Issue a batch of Credentials to enable the usage a unique Credential per presentation or per Verifier using Batch Credential Endpoint defined in (#batch-credential-endpoint). This only helps with Verifier/Verifier unlinkability.
+* Use cryptographic schemes that can provide non-correlation.
+
+### Credential Offer
+
+The Privacy Considerations in Section 11.2 of [@!RFC9101] apply to the `credential_offer` and
+`credential_offer_uri` parameters defined in (#credential-offer).
+
+### Authorization Request
+
+The Wallet SHOULD NOT include potentially sensitive information in the Authorization Request,
+for example, that including clear-text session information as a `state` parameter value, or encoding
+it in a `redirect_uri` parameter. A third party may observe such information through browser
+history, etc. and correlate the user's activity using it.
+
+## Credential Issuer Identifier
+
+The Credential Issuer Identifier alone may reveal information about the End-User.
```

Comment:
To whom would this information be revealed?

Reply:
The wallet, any verifier, and anyone else that is able to access the credential.

Reply:
AFAIK the OID4VCI spec does not explicitly state that the Credential Issuer Identifier must be stored in a credential, this is what confused me as I thought only the issuer and wallet would be the only parties privy to this information. See my below comment.

Reply:
@cobward typically a verifier requires knowing who the issuer is to some extend to establish trust, do you mean the concrete way the issuer is identified is different across credential formats?

Reply:
> do you mean the concrete way the issuer is identified is different across credential formats?

Yes my point was that the Credential Issuer Identifier isn't a concrete way that is used across all formats.

Reply:
that has been applied here; https://github.com/openid/OpenID4VCI/pull/244/files#r1480497886